### PR TITLE
Basic structure home

### PR DESCRIPTION
**Pull request by tanjahennis**
added the three reasons beneath the banner on the homepage

[](http://cd.sseu.re/20161026-3mdij.png)

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -14,8 +14,8 @@
   config.action_controller.perform_caching = false
 
   # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
-  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = { address: "localhost", port: 1025 }
```
Comment (miriamtocino) [inline config/environments/development.rb]: Why did you need to change this and how it is related to the basic structure of the home page?

File: app/assets/stylesheets/materialized-components/_redesign_page_header.sass [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        font-weight: 600
+        line-height: 56px
+      .subtitle
+        font-size: 40px
+        font-weight: 300
+        line-height: 56px
+
+
+
+
```
Comment (miriamtocino) [inline app/assets/stylesheets/materialized-components/_redesign_page_header.sass]: Remove these three extra lines. There should only be one extra line at the end of every file :wink:

Comment (miriamtocino) [inline app/assets/stylesheets/materialized-components/_redesign_page_header.sass]: Also, is this a new file? What is it related to? Only the homepage?

Did you talk to @durw4rd about how to organize all new `sass` files added for the new design? The folder `/materialized-components` doesn't look to me like a proper folder for this to live, but it might be a conscious decision :smile:

Maybe we could place all new components related to the `redesign` under `/materialized-components/redesign/_page_header.sass` and then import that folder into the main `sass` file.

Can you jump in @durw4rd and add your thoughts?

File: app/views/static_pages/home.html.slim [hunk 3]
```diff
@@ -2,69 +2,101 @@
...
+/            | Reviews via:
+/            =< link_to 'https://www.springest.nl/codaisseur-development-bootcamp#ervaringen' do
+/              = image_tag 'springest-logo.png', alt: 'Springest', style: 'height: 2.2em; margin-top: -1.2em'
+/
+/.dbc-content
+/  .dbc-container.dbc-quote
+/    .inner
+/      .row
+/        blockquote
+/          p = "“#{t('.quote')}”"
```
Comment (miriamtocino) [inline app/views/static_pages/home.html.slim]: Why are you leaving those commented lines in there? 

It's good practice to only commit files when they are clean and ready, so no lines commented :)

Comment (tanjahennis) [inline app/views/static_pages/home.html.slim]: They were there as a small guide for me to see what the original looked like, but indeed I should have removed it

File: app/assets/stylesheets/materialized-components/_redesign_page_header.sass (hunk 2 above)
Comment (tanjahennis) [inline app/assets/stylesheets/materialized-components/_redesign_page_header.sass]: I removed the lines :)

File: app/views/static_pages/home.html.slim (hunk 3 above)
Comment (tanjahennis) [inline app/views/static_pages/home.html.slim]: I removed the comments :)

Comment (durw4rd) [inline app/views/static_pages/home.html.slim]: we decided to just use one 'materialized-components' folder to store all new stylesheets for now, just avoid merge conficts within the application.css file
it's actually following what i saw in Codaisseur app :)

Comment (tanjahennis) [inline app/views/static_pages/home.html.slim]: I have updated the images to the corresponding black ones - they are in the images/illustrations folder and removed the yellow ones

File: config/environments/development.rb (hunk 1 above)
Comment (tanjahennis) [inline config/environments/development.rb]: We all did this - apart from michael, should we revert?

File: app/views/static_pages/home.html.slim (hunk 3 above)
Comment (miriamtocino) [inline app/views/static_pages/home.html.slim]: Awesome! Clear for me now :smile:

File: config/environments/development.rb (hunk 1 above)
Comment (tanjahennis) [inline config/environments/development.rb]: reverted by running gem uninstall mailcatcher

**Comment by miriamtocino** [comment]
We decided to close this pull request without merging, since all changes are also happening in #144 :smile:
